src/localization/bundles/en.json

```diff
@@ -698,6 +699,10 @@
     "server.deploy.progress.delete_network_agents.hasfailed": "Deleting network agents for compute host {0} has completed with partial failure. Followings are failed - \n\n{1}. \n\nDo you still want to proceed?",
     "server.deploy.progress.delete_network_agents.failure": "Error: Failed to delete network agents for compute host {0}. {1}",
     "server.deploy.installos.failure": "Failed to install SLES OS on compute host {0}.",
+    "server.deploy.retry.confirm": "Retry will rerun the failed playbook. Please examine the errors in the log and fix the issues before click Yes.  \n\n Do you want to continue?",
+    "server.deploy.failure.cancel.confirm": "Please examine the errors in the log and fix the issues encountered. \nPlease check the document's trouble shooting section on how to recover and try again. \n\n Do you want to continue?",
+    "server.addserver.failure.cancel.confirm": "Please examine the errors in the log and fix the issues encountered. Please do the following steps manually on the cloud deployer before click Yes to continue... \n1. Remove the new compute server(s) from ~/openstack/my_cloud/definition/data/servers.yml. \n2. git commit -a -m 'commite message'. \n3. ~/openstack/ardana/ansible/ansible-playbook -i hosts/localhost config-processor-run.yml \n4. ~/openstack/ardana/ansible/ansible-playbook -i hosts/localhost ready-deployment.yml  \n\nOnce it goes back the Add Server page, refresh page if it is necessary. Find servers at Manual Entry tab, update the servers and drag them to the right side table to retry deployment. \n\nPlease check the document's trouble shooting section for more details. \n\n Do you want to continue?",
```

Comment:
I'm confused... the user manually cancelled, but we're suggesting they check for errors in the log? Also we need to clean up the wording of some of this "before click Yes" is in the wrong tense (likely should be "before clicking Yes". I would remove the word 'compute' from "Remove the new compute server" , there's a typo "commite message" should be "commit message".  I dont understand what `Once it goes back the Add Server page, refresh page if it is necessary`  is trying to tell me to do. Also `Find servers at Manual Entry tab`  is confusing

Reply:
The cancel button is the not the cancel button that user manually cancel (in the middle). The cancel button is the the cancel button that shows up along with next button when error happens.  I will update the wordings.

Reply:
May need some editorial review from others, but as discussed the following may work: ```Check the log for errors and if there were any issues, resolve them before proceeding. The following steps need to be performed manually on the cloud deployer before clicking Yes to continue... \n1. Remove the new compute server(s) from ~/openstack/my_cloud/definition/data/servers.yml . \n2. git commit -a -m 'commite message'. \n3. ~/openstack/ardana/ansible/ansible-playbook -i hosts/localhost config-processor-run.yml \n4. ~/openstack/ardana/ansible/ansible-playbook -i hosts/localhost ready-deployment.yml . \n\n If the Add Server page continues to display the removed
servers, refresh the page using the web browser. To retry deployment, re-add the servers to the appropriate role, and click the "Deploy" button. \n\n Do you want to continue?```

Reply:
Also as discussed, we can add the bit about the troubleshooting guide back in once there is a troubleshooting section in the corresponding docs

Reply:
@jeremy-moffitt  thanks!